performance-storage-service/test.txt

```diff
@@ -0,0 +1 @@
+Trigger build
```

Comment:
@benliangw Can you delete this file next time you commit changes to test. I don't think it matters if this goes to staging but we should probably clean it up when we have a chance